.readthedocs.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
```

Comment:
This *may* fail, as I don't think RTD supports Python 3.11 runners yet?
I think the latest is 3.9. But we can test it if you want to?

Reply:
I refer [this](https://docs.readthedocs.io/en/stable/config-file/v2.html).